app/cmd/plugin_search.go

```diff
@@ -81,45 +84,60 @@ func matchPluginsData(plugins []client.AvailablePlugin) (result []client.CenterP
 	}
 	getCurrentJenkinsAndClient(&(jclient.JenkinsCore))
 	site, err := jclient.GetSite()
+	hasSite := false
 	if err != nil {
-		return
+		hasSite = true
```

Comment:
It's weird. Why if there are some errors then the hasSite is true?

Reply:
Maybe the name is not exact